src/main/java/com/opentable/metrics/JettyServerMetricsConfiguration.java

```diff
@@ -42,6 +42,12 @@
 public class JettyServerMetricsConfiguration {
     private static final String PREFIX = "http-server";
 
+    /**
+     * Create/expose a queued thread pool to use for the Jetty request pool
+     * @param metricRegistry the metric registry to use for recording thread pool metrics
+     * @param qSize the size of the the thread pool. Configured by "ot.httpserver.queue-size", defaults to 32,768 - THIS IS IGNORED
+     * @return the queued thread pool to use for requests
+     */
     @Bean
     public Provider<QueuedThreadPool> getIQTPProvider(final MetricRegistry metricRegistry, @Value("${ot.httpserver.queue-size:32768}") int qSize) {
```

Comment:
comment has nothing to do with the current PR, just a gripe: This bothers me to no end, why oh why are we configuring Jetty in Metrics lib :-(

Reply:
I'll create a task to move it.